deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-ocm-agent-alerts
+    role: alert-rules 
+  name: sre-ocm-agent-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-ocm-agent-alerts
+    rules:
+    - alert: KubePersistentVolumeFillingUp
+    #
+      expr: (kubelet_volume_stats_available_bytes{job="kubelet",metrics_path="/metrics"} / kubelet_volume_stats_capacity_bytes{job="kubelet",metrics_path="/metrics"}) < 0.03 and kubelet_volume_stats_used_bytes{job="kubelet",metrics_path="/metrics"} > 0 unless on(namespace, persistentvolumeclaim) kube_persistentvolumeclaim_access_mode{access_mode="ReadOnlyMany"} == 1 unless on(namespace, persistentvolumeclaim) kube_persistentvolumeclaim_labels{label_alerts_k8s_io_kube_persistent_volume_filling_up="disabled"} == 1
+      for: 1m
```

This does not need to be the same content as the `KubePersistentVolumeFillingUp`. It needs to be an alert that fires when `KubePersistentVolumeFillingUp` fires for the `openshift-logging` namespace.

Something like (please test this..)

```suggestion
      expr: ALERTS{alertname="KubePersistentVolumeFillingUp", alertstate="firing", namespace="openshift-logging"} == 1
      for: 30m
```

---

Fixed